SConstruct

```diff
@@ -106,23 +106,15 @@ linker_to_flags = {
   'clang++'  : clang_linker_flags
 }
 
-
-def cuda_installation():
+def cuda_installation(env):
```

Comment:
Thanks for pointing that out. I see now that the other like functions such as `tbb_installation` take an `env`, so `omp_installation` is really the odd one out. So I think your change to `cuda_installation` is OK.